FizzBuzz/FizzBuzz.cs

```diff
@@ -6,7 +6,29 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //      3-Fizz
+            //      5-Buzz
+            //      15-FizzBuzz
+
+            for (int i = 0; i < 300; i++)
```

Comment:
this will go from 0 to 299, it was supposed to start at 1 and go to the last number.